maintainers-guide.md

```diff
@@ -0,0 +1,210 @@
+   and PRs.
+   
+When an issue or PR is routed to the correct owner, the owner can address and
+prioritize the issue accordingly.
+
+The Developer Experience (DevEx) subteam handles the initial routing of issues
+and PRs on weekly rotation basis. The subteam can be reached on GitHub using the
+`@bazelbuild/devex` alias.
+
+## My team owns a label. What should I do?
+
+Subteams need to triage all issues in the [labels they own](), preferable on a weekly basis.
+
+### Issues
+
+1. Filter the list of issues by your team label **and** the `untriaged` label.
+1. Review the issue.
+1. Identify a priority level and assign the label.
+  1. The issue may have already been prioritized by the DevEx subteam if it's a P0. Re-prioritize if needed.
+1. Remove the `untriaged` label.
+
+
+Note that you need to be in the [bazelbuild
+organization](https://github.com/bazelbuild) to be able to add or remove labels.
+
+### Pull Requests
+
+1. Filter the list of pull requests by your team label.
+1. Review open pull requests.
+  1. **Optional**: If you are assigned for the review but is not the right fit
+  for it, re-assign the appropriate reviewer to perform a code review.
+1. Work with the pull request creator to complete a code review.
+1. Approve the PR.
+1. Ensure that all tests pass.
+1. Import the patch to the internal version control system and run the internal
+   presubmits.
+1. Submit the internal patch. If the patch submits and exports successfully, the
+   PR will be closed automatically by GitHub.
+
+## Priority
+
+The following definitions for priority will be used by the maintainers to triage
+issues.
+
+* **P0** - Major broken functionality that causes a Bazel release (minus release
+  candidates) to be unusable, or a downed service that severely impacts
+  development of the Bazel project. This includes regressions introduced in a
+  new release that blocks a significant number of users, or an incompatible
+  breaking change that was not compliant to the [Breaking
+  Change](https://docs.google.com/document/d/1q5GGRxKrF_mnwtaPKI487P8OdDRh2nN7jX6U-FXnHL0/edit?pli=1#heading=h.ceof6vpkb3ik)
+  policy.
+* **P1** - Critical feature request or bug fix which should be addressed in the
+  next release, or a serious issue that impacts many users. Typically does not
+  require immediate action. High demand, planned in the roadmap for the current
+  quarter.
+* **P2** - Important feature or defect that is tied to roadmaps or upcoming
+  releases. Moderate live issue in a released Bazel version that is inconvenient
+  for a user that needs to be addressed in a upcoming release.
+* **P3** - Desirable enhancement or minor bug fix that will benefit a user. Not
+  prioritized into Bazel roadmaps or any imminent release.
+* **P4** - Low priority defect or feature request that is unlikely to get
+  closed. Can also be kept open for a potential re-prioritization if more users
+  are impacted.
+
+## Team labels
```

Comment:
Shall have a list of people here too? (or at least one point of contact for each team)

Reply:
I'm not sure about this. I feel that issues and PRs should be funneled by the devex team to reduce toil and increase routing accuracy.

Reply:
Yes, I agree with @aiuto. @laurentlb, wdyt about the CODEOWNERS approach?

Reply:
For background. https://help.github.com/articles/about-codeowners/

Reply:
We already do this, just need to update the correct mapping. https://github.com/bazelbuild/bazel/blob/master/CODEOWNERS

Reply:
I still think it's important to have contact points for each area, e.g. a "TL". Issues have a team label. I want to know where to escalate, for example if an issue is not correctly triaged.

CODEOWNERS are for code paths, that seems to be something different.

Reply:
What about team aliases, like `@bazelbuild/devex` and `@bazelbuild/android`? (These two already exist)

Reply:
e.g. @bazelbuild/devex

Reply:
That would be good, except that the list seems private?

I think my preference is to keep the points of contact in the roadmaps (or another page), e.g.
https://www.bazel.build/roadmaps/android.html
and make sure each team has a roadmap

Reply:
yes, this is not meant for contributors. I also agree that the roadmaps (and maybe the contributor's page) should have direct points of contact.